monai/metrics/confusion_matrix.py

```diff
@@ -92,21 +96,10 @@ def __call__(self, y_pred: torch.Tensor, y: torch.Tensor):
 
         if self.compute_sample:
             confusion_matrix = compute_confusion_matrix_metric(self.metric_name, confusion_matrix)
-            if self.output_class:
-                f, not_nans = do_metric_reduction(confusion_matrix, MetricReduction.MEAN_BATCH)
-            else:
-                f, not_nans = do_metric_reduction(confusion_matrix, MetricReduction.MEAN)
+            f, not_nans = do_metric_reduction(confusion_matrix, self.reduction)
```

Comment:
could you add support for multiple metrics here? for example `metric_name=("sensitivity", "specificity")` it should return `f_0, not_nans_0, f_1, not_nans_1`

Reply:
Hi @wyli ,

Can we support this feature in `metrics` but not in `metrics ignite handler`?
If returning many items, it's not easy to support distributed data parallel.

Thanks.

Reply:
yes, for now let's do it for the metrics. thanks!